client/src/common/components/Header.tsx

```diff
@@ -1,9 +1,162 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import { Link } from 'react-router-dom';
+// import { FaTimes } from 'react-icons/fa';
+// import { FaBars } from 'react-icons/fa';
 
 const Header = () => {
-	// test tailwind
-	return <div className="bg-header_color text-white">This is Header</div>;
+	const [isClosed, setClosed] = useState(true);
 
+	useEffect(() => {
+		document.addEventListener('mousedown', () => {
+			setClosed(true);
+		});
+	});
+
+	const mymenu = React.useRef() as React.MutableRefObject<HTMLDivElement>;
+
+	return (
+		<div
+			className="bg-header_color
+						text-white flex
+						justify-between
+						items=center 
+						font-black 
+						w-screen
+						h-14
+						md:h-16
```

Comment:
```suggestion
						h-mobileHeaderHeight
						md:h-laptopHeaderHeight
```
We set a custom class in `tailwind.config.js` together, so let's use them here.